RockenRoller.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "RockenRoller",
```

Comment:
Change name to "Rock 'n Roller"? File name change to rocknroller.json too? I'm guessing that's what you meant.

Reply:
I want the e, like the name's supposed to be rocking roller or something like that.